fastlane/Fastfile

```diff
@@ -414,8 +414,8 @@ platform :android do
     android_firebase_test(
       project_id: firebase_secret(name: 'project_id'),
       key_file: GOOGLE_FIREBASE_SECRETS_PATH,
-      model: 'Nexus5',
-      version: 23,
+      model: 'Pixel2.arm',
+      version: 30,
```

Comment:
Updated it to use the same Emulator/OS combination [we've been using for other apps](https://github.com/wordpress-mobile/WordPress-Android/blob/e7e332d86cc528c996f2c539ff9d47e151ad03e8/fastlane/lanes/test.rb#L32C15-L32C25).